app/views/play/level/LevelGoalsView.coffee

```diff
@@ -53,7 +53,7 @@ module.exports = class LevelGoalsView extends CocoView
     @previousGoalStatus ?= {}
     @succeeded = e.overallStatus is 'success'
     for goal in e.goals
-      state = e.goalStates[goal.id]
+      state = e.goalStates[goal.id] or { status: 'incomplete' }
```

Comment:
Leaving this in as a way to avoid errors when the play view is in a transitioning state as the session is updated and the player has not moved on to the cinematic yet.